pom.xml

```diff
@@ -84,7 +84,7 @@ under the License.
 		<testRetryCount>3</testRetryCount>
 
 		<!-- Use latest Pulsar version -->
-		<pulsar.version>2.8.1</pulsar.version>
+		<pulsar.version>2.10.0</pulsar.version>
 		<flink.version>1.13.3</flink.version>
```

Comment:
instead of working on the master branch
you should update the release-1.14 branch.

Reply:
Oh should we first test the result out on master ? I have another branch for release-1.14 which is almost the same as master. Currently I'm stuck in a CI issue. Basically the test fail on local terminal and github runner, but succeed on Yufan's machine and my local intellij environment. I'll try another method if I cannot resolve the issue today.